.erb-lint.yml

```diff
@@ -50,8 +50,6 @@ linters:
           - 'btn(-(small|big|narrow|wide|link|primary|secondary|danger|disabled|big|narrow|transparent|border))?'
           - '(border|bg)-(none|black|gray|silver|aqua|blue|navy|teal|green|olive|lime|orange|red|fuchsia|purple|maroon|darken-[1-4]|lighten-[1-4])'
           - 'bg-(cover|contain|center|top|right|bottom|left)'
-          - 'h([013]|00)'
-          - 'h([01]|00)-responsive'
```

Comment:
These _are_ BassCSS ([source](https://github.com/basscss/basscss-sass/blob/48212a612efaa913599bbd273724622a00e0a7f4/scss/_type-scale.scss#L14-L26)), but at least by duplicating them into our stylesheets, we can still eliminate the BassCSS files associated with them. We may need to revisit this at some point in the future to align our headings with the design system, but that'll be a bigger issue to solve.